src/com/customify/client/services/ProductService.java

```diff
@@ -0,0 +1,58 @@
+package com.customify.client.services;
+
+import com.customify.client.Common;
+import com.customify.server.models.ProductModel;
```

Comment:
It is not a good practice for a client to directly access backend. Note that the communication should be a `tcp`

Reply:
Ok, I will create a product format